### PR TITLE
Nice new reworked Ghost Cafe aka alternative to Forrest's one

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Adds a remastered ghost cafe with on a new Z level, equipped with testing range, pacifism barriers and building zone

## Why It's Good For The Game

Safeties. We need more of them. (Also building stuff is fun)

## Changelog
:cl:
add: Added a new Ghost Cafe
add: Added a new Ghost Z level
add: Added a unique protolathe for the ghost cafe. It has a fully researched techweb but is bubblewrapped and disallowes you to communicate with station in any ways
/:cl:

### CODE REVIEW

**Comment by KrabSpider** [comment]
Screenshots would be pretty helpful

**Comment by Putnam3145** [comment]
@KrabSpider https://github.com/Citadel-Station-13/Citadel-Station-13/pull/13312/checks?check_run_id=1047717712

Anyway, this particular bit a big fat "no" from me, remove it
image

chemistry is too easy to make things unlivable with.

**Comment by davidchan13** [comment]
Honestly don't really care if people are killing each other in the ghost cafe, you can always respawn. If someone is explicitly griefing, bannu.

**Comment by Detective-Google** [comment]
image
 also probably a no

File: code/modules/research/machinery/protolathe.dm [hunk 1]
```diff
@@ -32,3 +32,38 @@
...
+	allowed_buildtypes = PROTOLATHE | IMPRINTER
+/obj/machinery/rnd/production/protolathe/offstation/ghost_cafe/Initialize(mapload)
+	. = ..()
+	host_research = SSresearch.admin_tech
+
+/obj/effect/landmark/ghost_protolathe
+	name = "Ghost Cafe Protolathe"
+	icon = 'icons/obj/machines/research.dmi'
+
+/obj/effect/landmark/ghost_protolathe/Initialize()
```
Comment (Detective-Google) [inline code/modules/research/machinery/protolathe.dm:66]: adding a landmark to spawn a protolathe instead of just placing one seems ??? to me

**Comment by Putnam3145** [comment]
okay unironically gonna close this unless the chemistry's removed, there is no justifiable reason to have it except exploits, i'm not joking, every single possible reason to have chemistry in the ghost cafe I can think of is an exploit

**Comment by davidchan13** [comment]
@putnam3145 Chemistry is already in the ghost cafe? And it's there for people to practice chemistry in an environment that won't negatively affect the round? Try new ratios, fermichems and seeing what methods work best for the desired result?

**Comment by Putnam3145** [comment]
No, there is no chem dispenser in the ghost cafe as of now, and adding one opens the way to an absolutely ridiculous amount of exploits. You can make the ghost cafe completely unusable without issue by, for example, nitrous oxide flooding. There is far too much that chemistry allows to allow a walled garden to have it.

Similarly, botany should at least be hugboxed not to have any way to make pod people.

EDIT: As an example, I think I saw a way to get uranium up there. Uranium allows you to make SDGF, which allows you to make a clone which would be capable of leaving the ghost cafe.

**Comment by davidchan13** [comment]
image

Yes, ghost cafe has a chem dispenser already in it's current form, sheet stacker spawns uranium. If someone is seeking out exploits to escape or grief the cafe, ban them and encourage players to ahelp it. 

There's no reason to kneecap it when it hasn't proved to be a problem thus far. 

If people want to build or play with chems to practice for later or or make something goofy for RP let them.

**Comment by Putnam3145** [comment]
That does not resemble any part of the ghost cafe on Citadel. This is the cit ghost cafe.
image

**Comment by DeltaFire15** [comment]
It has been some time and maintainer consensus is to not merge this. Closing this PR.

**Pull request closed without merging**